src/main/java/io/github/skyblockcore/mixin/PlayNetworkHandlerMixin.java

```diff
@@ -0,0 +1,67 @@
+package io.github.skyblockcore.mixin;
+
+import com.google.common.collect.Sets;
+import io.github.skyblockcore.SkyblockCore;
+import io.github.skyblockcore.event.JoinSkyblockCallback;
+import io.github.skyblockcore.event.LeaveSkyblockCallback;
+import io.github.skyblockcore.event.LocationChangedCallback;
+import io.github.skyblockcore.util.TextUtils;
+import net.minecraft.client.network.ClientPlayNetworkHandler;
+import net.minecraft.client.world.ClientWorld;
+import net.minecraft.network.packet.s2c.play.ScoreboardObjectiveUpdateS2CPacket;
+import net.minecraft.scoreboard.Scoreboard;
+import net.minecraft.scoreboard.ScoreboardObjective;
+import net.minecraft.scoreboard.ScoreboardPlayerScore;
+import net.minecraft.scoreboard.Team;
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Collection;
+import java.util.Set;
+
+@Mixin(ClientPlayNetworkHandler.class)
+public class PlayNetworkHandlerMixin {
+
+    @Shadow private ClientWorld world;
+    private static final Set<String> SKYBLOCK_TITLES = Sets.newHashSet("SKYBLOCK", "\u7A7A\u5C9B\u751F\u5B58", "\u7A7A\u5CF6\u751F\u5B58");
```

Comment:
Me when no SKIBLOCK

Reply:
In all seriousness, if this problem would have to be prevented we would have to come up with a way to detect when the player is on Skyblock without using the sidebar as they might as well change it to "skiblocki" or something, next year.

Reply:
I think it wouldn't make sense if you did it any other way, since I've had issues with all other methods
Even tho SKIBLOCK and other variations are annoying, if we code this correctly we will be able to fix it the moment the update happens 

NEU and other mods did this for the 2023 joke within like 2-5 hours

Reply:
I might code smth small for if it detects the nether star but not "SKYBLOCK" to still work or smth 
Idk on a bus right now :D

Reply:
The Skyblock Menu Item could be a viable option, all though you would have to account for:
1. Quiver arrows can also be placed there (not sure if when you join a new island / lobby holding a bow if it will first set the slot to the menu item and then to the quiver arrow, or just immediately the arrows).
2. There's some other cases where there would be a different item there. The only example I could think of right now is the Dojo minigames.

But to be honest, like you said, I feel like this isn't that big of an issue and can most likely be left as is.

Reply:
While both of those are true, we are talking about an edge-case that is only present on 1 day 

I also believe that the item is changed when you pick up M7 Part 5 Crystals 
and I'm not familiar with using any other method (Although I think skytils uses something else, don't quote me on this)

I'll look into how other mods detect if you're in skyblock/skiblock and see if something else is possible since in feature updates there might be a boss/item that corrupts your scoreboard (Like in murder mystery at the curse alter)

Reply:
> NEU and other mods did this for the 2023 joke within like 2-5 hours

Keep in mind that was with our slow release process. The actual fix to probably less than 10 minutes from hypixel changing the sidebar to the fix being in a pr.

Instead you could also just have a data repository and fetch from that a list of valid skyblock names. In general fetching data like that would be good, since this is intended to be a library mod, so it might be harder to update (since it needs to pass through at least 2 release cycles).